### PR TITLE
Multipointer input bindings

### LINKED ISSUE #766: Buttonpress event are fired on both controllers at the same time in XR
**For questions, please use the [forum](https://forum.babylonjs.com)**.

**Describe the bug**
Whatever controller (gamepad or hands) you use in XR, pointerdown events will be sent on both controllers at the same time to babylonjs.

### PR DESCRIPTION
**Pull request by CraigFeldspar**
Creates OpenXR binding for inputs for both controllers separately

Closes #766

### CODE REVIEW

File: Plugins/NativeXr/Source/NativeXr.cpp [hunk 1]
```diff
@@ -807,11 +807,16 @@ namespace Babylon
                 Napi::Function func = DefineClass(
                     env,
                     JS_CLASS_NAME,
-                    {});
+                    {Napi::ObjectWrap<PointerEvent>::InstanceAccessor("pointerId", &PointerEvent::GetPointerId, nullptr)});
```
Comment (rgerd) [inline Plugins/NativeXr/Source/NativeXr.cpp:810]:
```suggestion
                    {
                        InstanceAccessor("pointerId", &PointerEvent::GetPointerId, nullptr),
                    });
```

File: Plugins/NativeXr/Source/NativeXr.cpp [hunk 2]
```diff
@@ -820,7 +825,12 @@ namespace Babylon
             PointerEvent(const Napi::CallbackInfo& info)
                 : Napi::ObjectWrap<PointerEvent>{info}
             {
+                Napi::Object params = info[1].As<Napi::Object>();
+                pointerId = params.Get("pointerId").As<Napi::Number>().Int32Value();
             }
+
+        private:
+            int32_t pointerId;
```
Comment (rgerd) [inline Plugins/NativeXr/Source/NativeXr.cpp:833]: This is a member variable to it should be renamed to `m_pointerId`

**Review by rgerd — APPROVED** [review]
LGTM

**Comment by bghgary** [comment]
@CraigFeldspar Is this good to merge?

**Comment by CraigFeldspar** [comment]
@bghgary yep all good

**Pull request merged**